config/sync/field.field.node.full_width_banner_alert.field_alert_inheritance_subpages.yml

```
@@ -9,15 +9,15 @@ id: node.full_width_banner_alert.field_alert_inheritance_subpages
 field_name: field_alert_inheritance_subpages
 entity_type: node
 bundle: full_width_banner_alert
-label: 'Inherited by subpages'
-description: 'Should all pages of the VAMC display this block? For example, if you pick VA Pittsburgh health care, should va.gov/pittsburgh-health-care/about-us, and all other children pages, display this alert? '
+label: 'Only show on VAMC System Page & Operating Status Page'
+description: 'By checking this box, you will limit the alert to display only on the VAMC System Page and the Operating status page.'
```

Inconsistent capitalization 
```suggestion
description: 'By checking this box, you will limit the alert to display only on the VAMC System Page and the Operating Status Page.'
```

---

Updated, @swirtSJW